message/infrastructure/io/doc.go

```diff
@@ -0,0 +1,3 @@
+package io
```

Comment:
missing docs in watermill.io

Reply:
added docs. Lots of stuff, please review

Reply:
b3ae9f252ccb6e90efcf9f72512765a9df71cff4